### PR TITLE
SOV-4269: TVL endpoints

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-4269

Use `/legacy/cmc/tvl?chainId=xxx` to get TVL data as it's in old graph wrapper
Use `/legacy/cmc/tvl/summary` to get TVL data total values in USD and BTC summed for all supported chains

### CODE REVIEW

File: src/artifacts/graphQueries/legacy/token-prices.graphql [hunk 1]
```diff
@@ -1,6 +1,7 @@
 query ($ids: [ID]!) {
   tokens(where: { id_in: $ids }) {
     id
+    symbol
     lastPriceUsd
   }
 }
```
Comment (pietro-maximoff) [inline src/artifacts/graphQueries/legacy/token-prices.graphql]:
```suggestion
}

```

File: src/artifacts/abis/sdex/SdexSwapDex.json [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+          {
+              "internalType": "bytes",
+              "name": "",
+              "type": "bytes"
+          }
+      ],
+      "stateMutability": "payable",
+      "type": "function"
+  }
+]
```
Comment (pietro-maximoff) [inline src/artifacts/abis/sdex/SdexSwapDex.json]:
```suggestion
]

```

File: src/cronjobs/retrieve-usd-prices.ts [hunk 3]
```diff
@@ -95,8 +95,27 @@ async function prepareLegacyTokens(chain: LegacyChain, date: Date, tokensToQuery
       }
     }
 
+    // if ZUSD token price is requested, but subgraph does not return it, add price of DLLR to it.
+    if (
+      tokensToQuery.find((item) => item.address.toLowerCase() === chain.config.zusdToken.toLowerCase()) &&
```
Comment (pietro-maximoff) [inline src/cronjobs/retrieve-usd-prices.ts]: you can create a func like we have in dap2 - `areAddressesEqual` and use it there

File: migrations/meta/_journal.json [hunk 4]
```diff
@@ -22,6 +22,13 @@
...
+    },
+    {
+      "idx": 3,
+      "version": "6",
+      "when": 1720099286658,
+      "tag": "0003_needy_lightspeed",
+      "breakpoints": true
     }
   ]
 }
```
Comment (pietro-maximoff) [inline migrations/meta/_journal.json:34]:
```suggestion
}

```

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]

File: migrations/meta/_journal.json (hunk 4 above)
Comment (creed-victor) [reply migrations/meta/_journal.json:34]: it's autogenerated file, so will leave it unchanged

**Pull request merged**